tests/cases/compiler/noUnusedLocals_selfReferenceInClassStaticBlock.ts

```diff
@@ -0,0 +1,37 @@
+// @strict: true
+// @target: esnext
+// @noUnusedLocals: true
+// @noEmit: true
+
+// https://github.com/microsoft/TypeScript/issues/57620
+
+let BluethootDevice_create;
+class BluethootDevice {
```

Comment:
Happy to see that the way I always assumed Bluetooth is spelled is making it to the TS codebase :)